docs/v3/tutorials/server-and-worker.mdx

```diff
@@ -0,0 +1,199 @@
+   ```bash
+   helm repo add prefect https://prefecthq.github.io/prefect-helm
+   ```
+
+2. For a basic deployment with default values, using the latest Prefect version and no auth:
+   ```bash
+   helm install prefect-server prefect/prefect-server
+   ```
+
+3. For a customized deployment, create a values.yaml file for Prefect Server (see [values.yaml template](https://github.com/PrefectHQ/prefect-helm/blob/main/charts/prefect-server/values.yaml))
+
+4. Create a secret for the API basic auth:
+   ```bash
+   kubectl create secret generic server-auth-secret --from-literal auth-string='admin:password123'
+   ```
+
+5. Example `server-values.yaml` file for this tutorial:
+   ```yaml
+   server:
+     basicAuth:
+       enabled: true
+       existingSecret: "server-auth-secret"  # Must pre-exist if existingSecret is used; use authString otherwise
+     autoscaling:
+       enabled: true
+     livenessProbe:
+       enabled: true
+     readinessProbe:
+       enabled: true
+   ```
+
+6. Install Prefect Server with custom values:
+   ```bash
+   helm install prefect-server prefect/prefect-server -f server-values.yaml
+   ```
+
+   Expected output:
+   ```
+   NAME: prefect-server
+   LAST DEPLOYED: Tue Mar  4 09:08:07 2025
+   NAMESPACE: prefect-server
+   STATUS: deployed
+   REVISION: 1
+   TEST SUITE: None
+   NOTES:
+   Run the following command to port-forward the UI to your localhost:
+   $ kubectl --namespace prefect-server port-forward svc/prefect-server 4200:4200
+
+   Visit http://localhost:4200 to use Prefect!
+   ```
+
+7. Access the Prefect UI:
+   ```bash
+   kubectl --namespace prefect-server port-forward svc/prefect-server 4200:4200
+   ```
+
+8. Navigate to `localhost:4200` and login with `admin:password123`
+
+## Deploy a Prefect Worker
+
+To connect a worker to your Prefect server in the same cluster:
+
+1. Create a secret for the API basic auth:
+   ```bash
+   kubectl create secret generic worker-auth-secret --from-literal auth-string='admin:password123'
+   ```
+
+2. Create a `worker-values.yaml` file for this tutorial:
+   ```yaml
+   worker:
+     apiConfig: selfHostedServer
+     config:
+       workPool: kube-test
+     selfHostedServerApiConfig:
+       apiUrl: http://prefect-server.prefect-server.svc.cluster.local:4200/api
+       # apiUrl = <service>.<namespace> if in same cluster, or FQDN if external
+     basicAuth:
+       enabled: true
+       existingSecret: worker-auth-secret
+   ```
+
+3. Install the Prefect Worker:
+   ```bash
+   helm install prefect-worker prefect/prefect-worker -f worker-values.yaml
+   ```
+
+   Expected output:
+   ```
+   Release "prefect-worker" has been upgraded. Happy Helming!
+   NAME: prefect-worker
+   LAST DEPLOYED: Tue Mar  4 11:26:21 2025
+   NAMESPACE: prefect-server
+   STATUS: deployed
+   REVISION: 1
+   TEST SUITE: None
+   NOTES:
+   ```
+
+## Cleanup
+
+To uninstall Prefect Server and Worker:
+
+```bash
+helm uninstall prefect-server
+helm uninstall prefect-worker
```

Comment:
```suggestion
helm uninstall prefect-worker
helm uninstall prefect-server
```

What do you think of deleting them in reverse order? Probably doesn't really matter, but respects the dependencies and less likely to leave the worker in a confused state.